components/table/Table.tsx

```diff
@@ -948,7 +949,7 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
         className={classString}
         expandIconColumnIndex={expandIconColumnIndex}
         expandIconAsCell={expandIconAsCell}
-        emptyText={locale.emptyText}
+        emptyText={emptyText}
```

Comment:
1. `const { ... loading ... } = this.props;`
2. `emptyText={loading  && locale.emptyText}`

Reply:
fixed

2 should be `emptyText={!loading && locale.emptyText}`